wrappers/nodejs/index.js

```diff
@@ -3989,6 +3990,11 @@ const format = {
   * <br>16-bit per-pixel frame grabber format.
   */
   format_FG: 'FG',
+  /**
+  * String literal of <code>'Y411'</code>.
+  * <br>12-bit per-pixel. 6 pixel data stream taking 4 bytes.
+  */
+  format_Y411: 'Y411',
```

Comment:
Make uppercase? Same for `format_FG`...

Reply:
Done for Y411 and FG , all the formats above are lower case , I didn't change them I afraid to brake completability.